devicemaps.json

```diff
@@ -105,6 +105,27 @@
           }
         ]
       },
+      "90500-0151-G12": {
+        "description": "Madrid - 6 ethernet, 1 lte",
+        "alias": {
+          "vendor": "Silicom",
+          "sku": "90500-0151-G12"
```

Comment:
The vendor and sku here is supposed to reference another entry to inherit from. If you make this G11, you won't need to add the lte part.